R/produce_slf_ch_tests.R

```diff
@@ -0,0 +1,29 @@
+#' Care Home All Episodes Tests
```

Comment:
```suggestion
#' Care Home year specific Episodes Tests
```
Which script are you doing tests for here? There are already tests for the all care home episode file see the function `produce_sc_ch_episodes_tests` which is used in the script `extract_all_sc_ch_tests` however, there are no tests for the Care home year specific scripts which your scripts can be applied to